src/routes/blog/index.svelte

```diff
@@ -12,10 +28,23 @@
   import Spacing from '$lib/components/sections/spacing.svelte';
 
   export let posts: Newsletter[] = [];
+
+  async function filterPosts(event: CustomEvent) {
+    const tags = event.detail;
+
+    window.history.replaceState({}, '', `${window.location.pathname}?l=${tags.join(',')}`);
```

Comment:
Nit: `window` APIs are accessible without the `window` object.
```suggestion
    history.replaceState({}, '', location.pathname + '?l=' + tags.join(','));
```

Reply:
I fixed it.